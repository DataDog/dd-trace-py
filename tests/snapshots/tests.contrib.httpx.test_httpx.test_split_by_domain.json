[[
  {
    "name": "http.request",
    "service": "localhost:8001",
    "resource": "http.request",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "http",
    "meta": {
      "_dd.p.dm": "-0",
      "http.method": "GET",
      "http.status_code": "200",
      "http.url": "http://localhost:8001/status/200",
      "http.useragent": "python-httpx/x.xx.x",
      "runtime-id": "e0fc845aaa70412aa95b1bc36ab72982"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.measured": 1,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "system.pid": 85259
    },
    "duration": 5259000,
    "start": 1633032093749460000
  }]]
