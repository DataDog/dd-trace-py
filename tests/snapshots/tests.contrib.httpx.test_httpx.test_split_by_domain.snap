[[{"name" "http.request"
   "service" "localhost:8001"
   "resource" "http.request"
   "type" "http"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1620689782785813000
   "duration" 4578000
   "meta" {"runtime-id" "0975fd0faf2a4855b8e668f3a21af4d4"
           "http.method" "GET"
           "http.url" "http://localhost:8001/status/200"
           "http.status_code" "200"}
   "metrics" {"_dd.agent_psr" 1.0
              "_dd.measured" 1
              "_sampling_priority_v1" 1
              "system.pid" 4674
              "_dd.tracer_kr" 1.0}}]]
